test/verify/check-system-info

```diff
@@ -250,11 +250,22 @@ class TestSystemInfo(MachineCase):
         self.allow_journal_messages(
             "org.freedesktop.systemd1: couldn't get property org.freedesktop.systemd1.Service ExecMain at /org/freedesktop/systemd1/unit/systemd_2dtimedated_2eservice: GDBus.Error:org.freedesktop.DBus.Error.UnknownProperty: Unknown property")
 
-        self.login_and_go("/system")
+        self.login_and_go("/system", superuser=False)
+
+        b.wait_present('#system_information_systime_button[disabled]')
 
         # Make sure the date loads
         b.wait_text_not("#system_information_systime_button", "")
 
+        # Gain admin access
+
+        b.click(".pf-c-alert:contains('Web console is running in limited access mode.') button:contains('Turn on')")
+        b.wait_in_text(".modal-dialog:contains('Administrative access')", "Please authenticate")
+        b.set_input_text(".modal-dialog:contains('Administrative access') input", "foobar")
+        b.click(".modal-dialog button:contains('Authenticate')")
+        b.wait_not_present(".modal-dialog:contains('Administrative access')")
+        b.wait_not_visible(".pf-c-alert:contains('Web console is running in limited access mode.')")
```

Comment:
By now we have so many copies of this code that we should make two testlib.py helpers. But of course that can be a follow-up.